scripts/integration/test/CLI.test.js

```diff
@@ -601,4 +618,244 @@ describe( "Tests the CLI.", () => {
 
       expect( resolved1.account.metadata_uri ).toEqual( undefined );
    } );
+
+
+   it("Should create and transfer coins to an account escrow and a domain escrow", async () => {
+      const domain = `domain${Math.floor( Math.random() * 1e9 )}`;
+      const name = "test";
+      let unsigned = cli([ "tx", "starname", "register-domain", "--yes", "--domain", domain, "--from", signer, "--gas-prices", gasPrices, "--memo", memo(), "--generate-only" ])
+      let broadcasted = signAndBroadcastTx( unsigned);
+      expect( broadcasted.logs ).toBeDefined();
+
+      //TODO: uncomment when account implements TransferableObject again
```

Comment:
@LeCodeurDuDimanche , what's the status of TransferableObject?

Reply:
`TransferableObject` is an interface that objects must implement in order to be used in an escrow. The `Accounts` should not implement that interface for now because they should not be transferable, but as soon as we have creator domain type we can make accounts that belong to such domains trransferable

Reply:
You're right that accounts in a closed domain are not eligible.  However, accounts in an open domain should be able to be swapped (put in escrow).